### ISSUE TITLE
Running custom dataset

### ISSUE THREAD

**Issue by dansteiert** [BenevolentAI/DeeplyTough#15]
Hi,
I just started out with your tool and run into similar problems as others already reported and wanted to share my solutions, which worked for me.

I followed the installation instructions as indicated here: https://github.com/JoshuaMeyers/DeeplyTough#code-setup
- first issue was the installation of mdtraj, which could not be build the wheel for, instead of the frozen version I used the current version.
-  After setting up the installation and tried to run the custom dataset as indicated here: https://github.com/BenevolentAI/DeeplyTough#evaluation, I got an issue with the second conda environment, in deeplytough/misc/utils.py line 142 you used "source activate", my conda could not find the environment anymore, this I changed it to "conda activate"
- Third Issue arose somewhere in the mgltools library, which needs numpy.oldnumerics. As per installation I did not have numpy installed here, and found that the version 1.8.1 will fullfill those requirements.

Long story short:
change requirements.txt to the current version of mdtraj (1.9.7 currently)
install pip install numpy==1.8.1 after activating deeplytough_mgltools environment
change conda environment change call in deeplytough/misc/utils.py line 142 to "conda activate" (dependen on conda version (mine is 4.6)

I hope that this will help you and others facilitating the installation process.

**Comment by dansteiert** [comment]
After re-setting up my Linux machine(Ubuntu 20.04.3 LTS), I came across missing dependencies not incorporated into the setup

I propose the following:

```
# get necessary dependencies
sudo apt-get install curl
sudo apt install gcc g++ gfortran

# create new python 3 env and activate
conda create -y -n deeplytough python=3.6
conda activate deeplytough

# install legacy version of htmd from source
curl -LO https://github.com/Acellera/htmd/archive/refs/tags/1.13.10.tar.gz && \
    tar -xvzf 1.13.10.tar.gz && rm 1.13.10.tar.gz && cd htmd-1.13.10 && \
    python setup.py install && \
    cd .. && \
    rm -rf htmd-1.13.10;

# install remaining python3 reqs
apt-get -y install openbabel
pip install --upgrade pip && pip install -r requirements.txt && pip install --ignore-installed llvmlite==0.28

# install legacy se3nn library from source
git clone https://github.com/mariogeiger/se3cnn && cd se3cnn && git reset --hard 6b976bea4ea17e1bd5655f0f030c6e2bb1637b57 && mv experiments se3cnn; sed -i "s/exclude=\['experiments\*'\]//g" setup.py && python setup.py install && cd .. && rm -rf se3cnn
git clone https://github.com/AMLab-Amsterdam/lie_learn && cd lie_learn && python setup.py install && cd .. && rm -rf lie_learn

# create python2 env used for protein structure preprocessing
conda create -y -n deeplytough_mgltools python=2.7
# Add numpy dependency
conda activate deeplytough_mltools
pip install numpy==1.8.1
conda install -y -n deeplytough_mgltools -c bioconda mgltools=1.5.6
```

**Comment by Lizeerd** [comment]
hi, after completed the code setup, when i try the scripts, it raised: HTMD:logging setup failed. do you know how to fix it?
